### PR TITLE
Add text files

### PR DESCRIPTION
**Pull request by Guillemdb**
Signed-off-by: Guillem Duran <guillem@sourced.tech>

### CODE REVIEW

File: GLOSSARY.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+## Abstract Syntax Tree
```
Comment (zurk) [inline GLOSSARY.md]: I think this file should be in `ml-core` only.

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+The official support channels, for both users and contributors, are:
+
+* GitHub [issues](https://github.com/src-d/ml/issues)\*
+* Slack: \#machine-learning room in the [source{d} Slack](https://join.slack.com/t/sourced-community/shared_invite/enQtMjc4Njk5MzEyNzM2LTFjNzY4NjEwZGEwMzRiNTM4MzRlMzQ4MmIzZjkwZmZlM2NjODUxZmJjNDI1OTcxNDAyMmZlNmFjODZlNTg0YWM)
+
+\*Before opening a new issue or submitting a new pull request, it's helpful to search the project - it's likely that another user has already reported the issue you're facing, or it's a known issue that we're already aware of.
+
+## How to Contribute
+
+Pull Requests \(PRs\) are the main and exclusive way to contribute to the official sourced.ml project. In order for a PR to be accepted it needs to pass a list of requirements:
```
Comment (zurk) [inline contributing.md]:
```suggestion
Pull Requests \(PRs\) are the main and exclusive way to contribute to the official sourced.ml.mining project. In order for a PR to be accepted it needs to pass a list of requirements:
```

**Review by zurk — CHANGES_REQUESTED** [review]

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+* GitHub [issues](https://github.com/src-d/ml/issues)\*
+* Slack: \#machine-learning room in the [source{d} Slack](https://join.slack.com/t/sourced-community/shared_invite/enQtMjc4Njk5MzEyNzM2LTFjNzY4NjEwZGEwMzRiNTM4MzRlMzQ4MmIzZjkwZmZlM2NjODUxZmJjNDI1OTcxNDAyMmZlNmFjODZlNTg0YWM)
+
+\*Before opening a new issue or submitting a new pull request, it's helpful to search the project - it's likely that another user has already reported the issue you're facing, or it's a known issue that we're already aware of.
+
+## How to Contribute
+
+Pull Requests \(PRs\) are the main and exclusive way to contribute to the official sourced.ml
+.mining project. In order for a PR to be accepted it needs to pass a list of requirements:
```
Comment (zurk) [inline contributing.md]: let's have the project name on the one line

**Review by zurk — APPROVED** [review]

File: contributing.md (hunk 3 above)
Comment (Guillemdb) [reply contributing.md]: sorry, auto line break from pycharm. I just corrected it.

**Comment by Guillemdb** [comment]
I'm adding here the same changes requested in [ml-core PR#2](https://github.com/src-d/ml-core/pull/2).

File: MAINTAINERS.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# MAINTAINERS
+
+Vadim Markovtsev [vadim@sourced.tech](mailto:vadim@sourced.tech) \(@vmarkovtsev\)
```
Comment (vmarkovtsev) [inline MAINTAINERS.md]: Order: Konst, you, me

File: README.md [hunk 5]
```diff
@@ -1,3 +1,18 @@
-# Source{d} Ml Mining
+# Source{d} ML Mining
+ [![Travis build status](https://travis-ci.com/src-d/ml-mining.svg)](https://travis-ci.com/src-d/ml-mining)
```
Comment (vmarkovtsev) [inline README.md]:
```suggestion
[![Travis build status](https://travis-ci.com/src-d/ml-mining.svg)](https://travis-ci.com/src-d/ml-mining)
```

File: README.md [hunk 6]
```diff
@@ -1,3 +1,18 @@
...
+ [![Code coverage](https://codecov.io/github/src-d/ml-mining/coverage.svg)](https://codecov.io/github/src-d/ml-mining)
```
Comment (vmarkovtsev) [inline README.md]:
```suggestion
[![Code coverage](https://codecov.io/github/src-d/ml-mining/coverage.svg)](https://codecov.io/github/src-d/ml-mining)
```

File: README.md [hunk 7]
```diff
@@ -1,3 +1,18 @@
...
+ [![Read the Docs](https://img.shields.io/readthedocs/ml-mining.svg)](https://readthedocs.org/projects/ml-mining/)
```
Comment (vmarkovtsev) [inline README.md]:
```suggestion
[![Read the Docs](https://img.shields.io/readthedocs/ml-mining.svg)](https://readthedocs.org/projects/ml-mining/)
```

**Review by vmarkovtsev — CHANGES_REQUESTED** [review]

**Review by vmarkovtsev — APPROVED** [review]

**Pull request merged**